src/safeposix/dispatcher.rs

```diff
@@ -351,18 +351,11 @@ pub extern "C" fn dispatcher(cageid: u64, callnum: i32, arg1: Arg, arg2: Arg, ar
             let nfds = get_onearg!(interface::get_int(arg1));
             if nfds < 0 { //RLIMIT_NOFILE check as well?
```

Comment:
it may make more sense to do this check in NaCl

Reply:
or move actually into the select_syscall, i think either of those options would be cleaner